system/mount.py

```diff
@@ -250,87 +344,138 @@ def unset_mount(module, **kwargs):
 
 
 def mount(module, **kwargs):
-    """ mount up a path or remount if needed """
+    """Mount up a path or remount if needed."""
 
-    # solaris kwargs: name, src, fstype, opts, boot, passno, state, fstab=/etc/vfstab
-    # linux kwargs: name, src, fstype, opts, dump, passno, state, fstab=/etc/fstab
+    # solaris kwargs:
+    #   name, src, fstype, opts, boot, passno, state, fstab=/etc/vfstab
+    # linux kwargs:
+    #   name, src, fstype, opts, dump, passno, state, fstab=/etc/fstab
     if get_platform() == 'SunOS':
```

Comment:
Given the multiple calls to get_platform, I'd probably just do a 'platform = get_platform()' and use 'platform' var through out.

And given there are multiple methods that need the platform info, I'd probably set it in the main and pass it to the methods. Especially if the methods are switched to use regular args and not passed a args/kwargs.